IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Functions/IfcAlignmentRepresentation/Expression.txt

```diff
@@ -0,0 +1,40 @@
+(Alignment : IfcAlignment) : IfcBoolean;
+	
+	LOCAL
+		AlignmentRepresentations : SET OF IfcShapeRepresentation := SELF.Representation.Representations;
+	END_LOCAL;
```

Comment:
I wonder how this will be initialized if `SELF.Representation` is `NIL`. (it is checked only after these lines)

Reply:
I have to look up how local variables are initialized. I did not think before initializing. 😄